src/cls/rbd/cls_rbd.cc

```diff
@@ -8294,12 +8319,13 @@ int rwlcache_get_cacheinfo(cls_method_context_t hctx, bufferlist *in, bufferlist
     return -EIO;
   }
 
-  if (map.caches.count(cache_id)) {
-    auto &cache = map.caches[cache_id];
+  if (map.uncommitted_caches.count(cache_id)) {
```

Comment:
For false-condition, it mean timeout and already removed. So it should return -ETIMEOUT.

Reply:
done